### PR TITLE
Prepare for testing

### PR DESCRIPTION
**Pull request by oshinowo-stephen**

### CODE REVIEW

File: .github/workflows/node.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v2
+      - name: building package manager
+        run: npm i && npm run build
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: testing package manager
+        run: npm i && npm run test
+  lint:
```
Comment (bartekleon) [inline .github/workflows/node.yml]: this step can be removed

File: tests/getVersion.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+
+import { getVersion } from '../src/getVersion';
+
+test.failing('get latest package version', async (t) => {
```
Comment (oshinowo-stephen) [inline tests/getVersion.ts:5]: the `getVersion` function without version name breaks...

File: package.json [hunk 3]
```diff
@@ -4,8 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "pretest": "export TS_NODE_FILES=true",
     "clean": "rimraf \"+src/**/*+(.d.ts|.js|.map)\"",
+    "test": "yarn pretest && nyc --reporter lcov ava",
```
Comment (bartekleon) [inline package.json]: remove "yarn pretest"

File: package.json [hunk 4]
```diff
@@ -4,8 +4,11 @@
...
     "prebuild": "npm run clean && eslint . --ext .ts",
+    "lint": "eslint src",
```
Comment (bartekleon) [inline package.json]: this can be removed

**Review by bartekleon — CHANGES_REQUESTED** [review]

File: tests/extractVersion.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  t.assert(majorDep, '1');
+});
+
+test('extract current ver', async (t) => {
+  const currentDep = extractVersion('1.0.0');
+
+  t.assert(currentDep, '1.0.0');
+});
+
+test.skip('extract invalid ver', async (t) => {
```
Comment (oshinowo-stephen) [inline tests/extractVersion.ts:23]: should also handle if the version inserted is invalid in some way?

File: package.json (hunk 3 above)
Comment (oshinowo-stephen) [reply package.json]: resovled

**Pull request merged**